packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -603,7 +603,7 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       installElasticAgent: `${FLEET_DOCS}install-fleet-managed-elastic-agent.html`,
       installElasticAgentStandalone: `${FLEET_DOCS}install-standalone-elastic-agent.html`,
       upgradeElasticAgent: `${FLEET_DOCS}upgrade-elastic-agent.html`,
-      upgradeElasticAgent712lower: `${FLEET_DOCS}upgrade-elastic-agent.html#upgrade-7.12-lower`,
+      upgradeElasticAgent712lower: `${FLEET_DOCS}upgrade-elastic-agent.html`,
```

Comment:
there is already a link to `upgradeElasticAgent` page in line 605, so `upgradeElasticAgent712lower` can be removed